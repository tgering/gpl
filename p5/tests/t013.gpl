// use M_PI from /usr/include/limits.h
// DO NOT CREATE YOUR OWN VALUE FOR PI

double a = sin(42.0);
double b = cos(42.0);
double c = tan(42.0);

double d = asin(a);
double e = acos(b);
double f = atan(c);

double g = asin(.5);
double h = acos(.5);
double i = atan(.5);

double j = sin(g);
double k = cos(h);
double l = tan(i);

double m = sqrt(144.0);
double n = abs(-42.0);
double o = abs(-42);
double p = abs(42);

double q = floor(42.42);
double s = floor(42);
double t = floor(.42);


