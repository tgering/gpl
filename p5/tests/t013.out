gpl.cpp::main()
  input file(tests/t013.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double a = 0.669131
double b = 0.743145
double c = 0.900404
double d = 42
double e = 42
double f = 42
double g = 30
double h = 60
double i = 26.5651
double j = 0.5
double k = 0.5
double l = 0.5
double m = 12
double n = 42
double o = 42
double p = 42
double q = 42
double s = 42
double t = 0
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
